src/Chapter1.hs

```diff
@@ -631,8 +642,13 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = (last2 n - last1 n) `div` 10 + last1 n
+    where
+        last2 :: Int -> Int
+        last2 n = abs n `mod` 100
```

Comment:
Nice work on using infix form of calling functions! We don't explain this syntax in our course to make it short. But well done :+1: